config/jobs/kubernetes/sig-release/release-branch-jobs/1.11.yaml

```diff
@@ -599,3 +627,48 @@ periodics:
           resources:
             requests:
               cpu: 4
+  # This is a sig-release-<stable3>-blocking job.
+  - cron: "0 8-20/12 * * *" # every six hours, disjoint with ci-kubernetes-e2e-gci-gce-scalability-stable1 and ci-kubernetes-e2e-gci-gce-scalability-stable2
```

Comment:
If this is similar to cron then this is an invalid comment.

I think we should just delete these comments. they do not seem to be maintained well. They could be misleading infact if someone does not know cron semantics and just reads the english to satisfy.